lib/CrashHandler/CrashHandler.cpp

```diff
@@ -685,6 +840,20 @@ void CrashHandler::installCrashHandler() {
 
     CrashHandler::crash(buffer.view());
   });
+
+  std::atexit([]() {
+    CrashHandler* ch = CrashHandler::_theCrashHandler;
```

Comment:
Maybe do a 
```suggestion
    auto* ch = CrashHandler::_theCrashHandler.exchange(nullptr);
```
instead? And similarly in `std::at_quick_exit` and `~CrashHandler()`. That could avoid races between `~CrashHandler()` and the exit handlers.